app/src/main/java/org/p2p/wallet/send/ui/main/SendPresenter.kt

```diff
@@ -615,11 +636,20 @@ class SendPresenter(
         setTargetResult(first)
     }
 
-    private fun updateButtonText(source: Token.Active) {
+    private fun updateButtonText(source: Token.Active, sendFee: FeeRelayerSendFee?) {
```

Comment:
sendFee: FeeRelayerSendFee? = null and do not make calls like
 updateButtonText(token, sendFee = null)

Reply:
im okay with explicit sense of telling that I pass fee as null